### DISCUSSION TITLE
History Api in js target

### DISCUSSION THREAD

**Discussion by JamshedAlamQaderi** [Tlaster/PreCompose#111] [Q&A] [answered]
All i want to know before using this awesome library that does it support history api in js target? like it will change browser url when i browse to another page and when i'm visiting from an url then via deeplink i should get the right page?

Thanks

**Comment by Tlaster** [comment]
Current does not support history api, but visiting from an url should get the right page.

**Reply by JamshedAlamQaderi** [reply]
how i could do that?

**Reply by Tlaster** [reply, marked as answer]
Not by built in but you can have a navigator and pass the route to it.

**Reply by JamshedAlamQaderi** [reply]
ohh. Now i got the idea...
Thank so much
